README.md

```diff
@@ -1,13 +1,15 @@
-# ice-pop
+# addepar-pop-menu
 
 Ember Addons for popping things up. This includes popovers, dropdowns, and tooltips.
 
-Documentation for the components are here: https://component-library.addedev.com/docs/css-docs/section-ice-pop.html
```

Comment:
Figured could pull this out since its severely out of date and not moving forward anymore.